test/services/editions/EditionsTemplatingTest.scala

```diff
@@ -0,0 +1,55 @@
+package services.editions
+
+import java.time.{LocalDate, ZonedDateTime}
+
+import com.gu.contentapi.client.model.v1.SearchResponse
+import fixtures.TestEdition
+import model.editions.{ArticleMetadata, CapiPrefillQuery, EditionsArticleSkeleton, Image, MediaType}
+import org.scalatest.{FreeSpec, Matchers, OptionValues}
+import services.{Capi, PrefillMetadata}
+
+import scala.concurrent.Future
+
+class EditionsTemplatingTest extends FreeSpec with Matchers with OptionValues {
+  "Creating a template" - {
+    "Sets the prefill metadata from CAPI" in {
+      val fakeCapi = new Capi {
+        def getPreviewHeaders(url: String): Seq[(String, String)] = ???
```

Comment:
do we need to add a concrete implementation here?

Reply:
no, it's not called in the test.